source/extensions/extensions_metadata.yaml

```diff
@@ -0,0 +1,579 @@
+envoy.access_loggers.file:
```

Comment:
I think you need to update developer docs to cover this file as well.

Reply:
yep - as @dmitri-d asked - i will add to the dev docs about updating this file